org.eclipse.scanning.api/src/org/eclipse/scanning/api/scan/ScanEstimator.java

```diff
@@ -89,6 +95,11 @@ public ScanEstimator(IPointGeneratorService pservice, ScanRequest<?> request) th
 	public ScanEstimator(IPointGeneratorService pservice, ScanRequest<?> request, long timePerPoint) throws GeneratorException {
 		this(pservice.createCompoundGenerator(request.getCompoundModel()), request.getDetectors(), timePerPoint);
 	}
+	
+	public ScanEstimator(Iterable<IPosition> positionIterable, List<IRunnableDevice<?>> detectors) throws GeneratorException {
+		this(positionIterable, detectors.stream().map(d -> d.getModel()).collect(Collectors.toList()), 0);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. (sonar.java.source not set. Assuming 8 or greater.) [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1612)